plaso/parsers/setupapi.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+"""Parser for Windows Setupapi log files."""
+
+from __future__ import unicode_literals
+
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import logger
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class SetupapiLogEventData(events.EventData):
+  """Setupapi log event data.
+
+  Attributes:
+    entry_type (str): log entry type such as "Device Install".
+    section_start (str): date and time of the start of the log entry event.
+    # message (str): contents of the log entry.
+    # section_end (str): date and time of the start of the log entry event.
+    exit_status (str): the exit status of the entry.
+  """
+
+  DATA_TYPE = 'setupapi:log:line'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(SetupapiLogEventData, self).__init__(data_type=self.DATA_TYPE)
+    self.entry_type = None
+    # TODO: Add fields
```

Comment:
Why are these commented out? Are you planning to add them as part of this PR or later?

If you are not planning to use this in PR, I opt to remove them and re-add them later.